packages/next-templates/src/app/styling/css/productpagina-klachtenformulier.css

```diff
@@ -0,0 +1,61 @@
+/* ----- COMPONENT STYLING ----- */
+
+/*  ALL COMPONENTS @ product pagina klachtenformulier 
+/*  Article, BreadcrumbNav, BreadcrumbNavLink, ButtonLink, ButtonGroup, Heading1, 
+/*    Heading2, Link, Paragraph, Page, PageContent, SpotlightSection, UnorderedList, 
+/*    UnorderedListItem, ExampleHeader, ExampleNavigation, ExampleFooter, IconChevronRight,
+/*   IconChevronUp
+ */
+
+/* GET STYLING FROM globals.css 
+/* PageContent, Paragraph, UnorderedList, ExampleHeader, ExampleFooter,
+/* IconChevronRight, IconChevronUp
+*/
+/* NEED (ADDITIONAL) STYLING IN globals.css */
+/* NEED STYLING REMOVED/CHANGES IN globals.css */
+/* GET STYLING FROM page.module.css */
+/* GET STYLING FROM detail-page.css 
+/* SpotlightSection 
+*/
+/* GET STYLING FROM SOMEWHERE ELSE (+ FROM WHERE) */
+/* NEED STYLING IN productpagina-klachtenformulier.css */
+
+/* ----- STYLING TO-DO'S ----- */
+
+/* BREADCRUMNAV */
+/* flex/fix chevron  */
+
+/* SPOTLIGHTSECTION */ /* styling from detail-page.css */
+/* needs padding top? */
+
+/* UNORDERED LIST */
+/* Klacht schriftelijk of telefonisch doorgeven: needs flex? is 6 rows in Figma */
+
+/* LINK */
+/* text + icon inline */
+
+/* naar boven link: on the bottom right of the page */
+/* naar boven link: should smooth scroll to top of page */
+
+/* .utrecht-breadcrumb-nav__list {
```

Comment:
waarom is dit uitgecomment?

Reply:
Het was me niet duidelijk hoe het zit met de styling, daarom had ik voor mezelf een opzetje gemaakt om overzicht te krijgen. Had in mn draft PR message gevraagd of mensen hier tips voor hadden. 

Heb hier inmiddels al een deel van weggehaald omdat ik er steeds meer van begrijp!💪🏼  
Die PR komt later, als ik een kopie van deze branch heb gemaakt en nog wat aanpassingen heb doorgevoerd!